<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>1510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>1100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>基于非配合人脸识别的智能课堂考勤系统</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>人脸注册</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_camera">
          <property name="minimumSize">
           <size>
            <width>660</width>
            <height>800</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>显示窗口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QPushButton" name="btn_open_camera">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>开启摄像头</string>
           </property>
          </widget>
          <widget class="QSplitter" name="splitter_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="splitter_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>课程名称</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_show_course">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
           <widget class="QPushButton" name="btn_create_data">
            <property name="text">
             <string>创建课程</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QFormLayout" name="formLayout">
            <property name="horizontalSpacing">
             <number>2</number>
            </property>
            <property name="verticalSpacing">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>姓名</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="text_classes">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>学号</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QTextEdit" name="text_num">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>班级</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTextEdit" name="text_name">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QPushButton" name="btn_insert">
           <property name="text">
            <string>添加成员</string>
           </property>
          </widget>
          <widget class="QSplitter" name="splitter_10">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QTextBrowser" name="text_path">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_path">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>浏览</string>
             </property>
            </widget>
           </widget>
           <widget class="QLabel" name="label_detect_output">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>保存预处理路径</string>
            </property>
           </widget>
           <widget class="QSplitter" name="splitter_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QTextBrowser" name="text_detect_path">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_detect_path">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>浏览</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_detect">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>图像预处理</string>
             </property>
            </widget>
           </widget>
           <widget class="QLabel" name="label_train_output">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>200</height>
             </size>
            </property>
            <property name="text">
             <string>训练分类器生成pkl文件,保存pkl路径</string>
            </property>
           </widget>
           <widget class="QSplitter" name="splitter_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QTextBrowser" name="text_train_path">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_train_path">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>浏览</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_train">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>注册</string>
             </property>
            </widget>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>人脸识别</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_camera_recog">
          <property name="minimumSize">
           <size>
            <width>660</width>
            <height>600</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>显示窗口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_camera_recog">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>开始识别</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>考勤结果</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QSplitter" name="splitter_13">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="splitter_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QLabel" name="label_10">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>课程</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_cla">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_12">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QLabel" name="label_8">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>课时</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_cla_time">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableView" name="table_data_view"/>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QPushButton" name="btn_Attendance">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>查询考勤数据</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_count">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_save_Attendance">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>保存考勤数据</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>说明界面</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_sys_close">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>关闭系统</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
